traveler-swift-core/TravelerKit/Models/PostPurchaseDetail.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  PostPurchaseDetail.swift
+//  TravelerKit
+//
+//  Created by Rakin Hoque on 2019-12-09.
+//  Copyright © 2019 Ata Namvari. All rights reserved.
+//
+
+import Foundation
+
+public struct PostPurchaseDetail: Decodable {
```

Comment:
Im assuming this is one of those temporary structs we're using to accomodate the current response from the backend. Otherwise I would need some walking through as to the importance of this model and why it exists. If it is a model we're using internally then it shouldn't be public. Also, if its temporary it should have a TODO comment with it mentioning the situation and that it will get deprecated eventually (or that it should be and that nobody should depend on that model...so technical debt doesnt pile due to the existence of this model).

Reply:
Moved into PurchasedParkingProduct as a private struct to remove reliance on this.  Added TODO as well.